modules/govuk/manifests/apps/rummager/rabbitmq.pp

```diff
@@ -34,23 +38,32 @@
     amqp_queue    => 'rummager_to_be_indexed',
     routing_key   => '*.links',
     durable       => true,
-  } ->
+  }
 
   govuk_rabbitmq::queue_with_binding { 'rummager_govuk_index':
     amqp_exchange => $amqp_exchange,
     amqp_queue    => 'rummager_govuk_index',
     routing_key   => '*.*',
```

Comment:
Will the bulk reindexing messages also be added to the existing queue, and is that OK? (IIRC we don't do any extra filtering by the routing key in rummager.)

Reply:
Nope, I deliberately made the routing key of the messages 3 parts to avoid this: eg `answer.bulk.reindex`.

Reply:
This is something that it might be worth documenting with a comment so that future generations can understand the process.